iogt/settings/base.py

```diff
@@ -534,3 +535,4 @@
 SUPERSET_PASSWORD = os.getenv('SUPERSET_PASSWORD')
 
 PUSH_NOTIFICATION = os.getenv('PUSH_NOTIFICATION', 'disable') == 'enable'
+JQUERY = os.getenv('JQUERY', 'disable') == 'enable'
```

Comment:
Disable jQuery by default?

Reply:
@istride JQuery is enabled by default. This expression compares the value of the JQUERY environment variable with the string 'enable'. If the value is 'enable', then the expression will evaluate to True. Otherwise, it will evaluate to False.